Userland/DevTools/UserspaceEmulator/Emulator.cpp

```diff
@@ -239,6 +248,104 @@ int Emulator::exec()
     return m_exit_status;
 }
 
+void Emulator::handle_REPL()
+{
+    // Console interface
+    // FIXME: Previous Instruction**s**
+    // FIXME: Function names (base, call, jump)
+    auto saved_eip = m_cpu.eip();
+    m_cpu.save_base_eip();
+    auto insn = X86::Instruction::from_stream(m_cpu, true, true);
+    // FIXME: This does not respect inlineing
+    //        another way of getting the current function is at need
+    if (const auto* region = load_library_from_adress(m_cpu.base_eip()).value_or(nullptr)) {
```

Comment:
`const auto*` -> `auto const*`

Reply:
I changed that for all things in both files now, although I dont really see the benefits of either convention....